build problems with electron 5 beta

Seeing this when running with latest electron 5 beta and v4.4.1:

```
C:\Users\SoloDolo\lbry-latest\lbry-desktop\node_modules\keytar>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
⡀
Error output:
prebuild-install WARN install No prebuilt binaries found (target=5.0.0-beta.8 runtime=electron arch=x64 libc= platform=win32)
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@10.15.3 | win32 | x64
gyp ERR! configure error```

I'm able to build from source against the desired runtime on Windows, which should be the fallback behaviour when there's no prebuilt binary for it:

```
 $ npm_config_runtime=electron npm_config_disturl=https://atom.io/download/electron npm_config_target=5.0.0-beta.8 npm_config_arch=x64 npm install


> keytar@4.4.1 install C:\Users\shiftkey\src\node-keytar
> prebuild-install || node-gyp rebuild


C:\Users\shiftkey\src\node-keytar>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  async.cc
  main.cc
c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(4458): warning C4996: 'v8::WasmModuleObject::SerializedModule': was declared deprecated (compiling source file ..\src\main.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(4380): note: see declaration of 'v8::WasmModuleObject::SerializedModule' (compiling source file ..\src\main.cc)
c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(4458): warning C4996: 'v8::WasmModuleObject::SerializedModule': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(4380): note: see declaration of 'v8::WasmModuleObject::SerializedModule' (compiling source file ..\src\async.cc)
c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(10172): warning C4996: 'v8::Value::ToBoolean': was declared deprecated (compiling source file ..\src\main.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(2387): note: see declaration of 'v8::Value::ToBoolean' (compiling source file ..\src\main.cc)
c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(10172): warning C4996: 'v8::Value::ToBoolean': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(2387): note: see declaration of 'v8::Value::ToBoolean' (compiling source file ..\src\async.cc)
c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\node_object_wrap.h(41): warning C4996: 'v8::PersistentBase<v8::Object>::IsNearDeath': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(571): note: see declaration of 'v8::PersistentBase<v8::Object>::IsNearDeath' (compiling source file ..\src\async.cc)
c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_converters_43_inl.h(22): warning C4996: 'v8::Value::ToBoolean': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(2387): note: see declaration of 'v8::Value::ToBoolean' (compiling source file ..\src\async.cc)
c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\node_object_wrap.h(41): warning C4996: 'v8::PersistentBase<v8::Object>::IsNearDeath': was declared deprecated (compiling source file ..\src\main.cc)c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_converters_43_inl.h(40): warning C4996: 'v8::Value::BooleanValue': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]

  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(571): note: see declaration of 'v8::PersistentBase<v8::Object>::IsNearDeath' (compiling source file ..\src\main.cc)c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(2432): note: see declaration of 'v8::Value::BooleanValue' (compiling source file ..\src\async.cc)

c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_converters_43_inl.h(22): warning C4996: 'v8::Value::ToBoolean': was declared deprecated (compiling source file ..\src\main.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(2387): note: see declaration of 'v8::Value::ToBoolean' (compiling source file ..\src\main.cc)
c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_converters_43_inl.h(40): warning C4996: 'v8::Value::BooleanValue': was declared deprecated (compiling source file ..\src\main.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(2432): note: see declaration of 'v8::Value::BooleanValue' (compiling source file ..\src\main.cc)
c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_object_wrap.h(24): warning C4996: 'v8::PersistentBase<v8::Object>::IsNearDeath': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(571): note: see declaration of 'v8::PersistentBase<v8::Object>::IsNearDeath' (compiling source file ..\src\async.cc)
c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_object_wrap.h(127): warning C4996: 'v8::PersistentBase<v8::Object>::IsNearDeath': was declared deprecated (compiling source file ..\src\async.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(571): note: see declaration of 'v8::PersistentBase<v8::Object>::IsNearDeath' (compiling source file ..\src\async.cc)
c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_object_wrap.h(24): warning C4996: 'v8::PersistentBase<v8::Object>::IsNearDeath': was declared deprecated (compiling source file ..\src\main.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(571): note: see declaration of 'v8::PersistentBase<v8::Object>::IsNearDeath' (compiling source file ..\src\main.cc)
c:\users\shiftkey\src\node-keytar\node_modules\nan\nan_object_wrap.h(127): warning C4996: 'v8::PersistentBase<v8::Object>::IsNearDeath': was declared deprecated (compiling source file ..\src\main.cc) [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(571): note: see declaration of 'v8::PersistentBase<v8::Object>::IsNearDeath' (compiling source file ..\src\main.cc)
c:\users\shiftkey\src\node-keytar\src\async.cc(193): warning C4996: 'v8::Object::Set': was declared deprecated [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(3259): note: see declaration of 'v8::Object::Set'
c:\users\shiftkey\src\node-keytar\src\async.cc(194): warning C4996: 'v8::Object::Set': was declared deprecated [C:\Users\shiftkey\src\node-keytar\build\keytar.vcxproj]
  c:\users\shiftkey\.node-gyp\5.0.0-beta.8\include\node\v8.h(3259): note: see declaration of 'v8::Object::Set'
  keytar_win.cc
  win_delay_load_hook.cc
     Creating library C:\Users\shiftkey\src\node-keytar\build\Release\keytar.lib and object C:\Users\shiftkey\src\node-keytar\build\Release\keytar.exp
  Generating code
  All 383 functions were compiled because no usable IPDB/IOBJ from previous compilation was found.
  Finished generating code
  keytar.vcxproj -> C:\Users\shiftkey\src\node-keytar\build\Release\\keytar.node
audited 1967 packages in 9.016s
found 38 vulnerabilities (1 low, 32 moderate, 5 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

--- a/package.json
+++ b/package.json
@@ -46,7 +46,7 @@
     "babel-plugin-transform-async-to-generator": "^6.24.1",
     "chai": "^4.2.0",
     "mocha": "^6.1.4",
-    "node-abi": "^2.7.0",
+    "node-abi": "^2.8.0",
     "node-cpplint": "~0.4.0",
     "node-gyp": "^4.0.0",
     "prebuild": "^8.2.1"

--- a/package.json
+++ b/package.json
@@ -36,8 +36,8 @@
     "test": "npm run lint && npm build . && mocha --require babel-core/register spec/",
     "prebuild-node": "prebuild -t 6.11.0 -t 7.9.0 -t 8.9.0 -t 9.4.0 -t 10.11.0 -t 11.9.0 -t 12.0.0 --strip",
     "prebuild-node-ia32": "prebuild -t 6.11.0 -t 7.9.0 -t 8.9.0 -t 9.4.0 -a ia32 --strip",
-    "prebuild-electron": "prebuild -t 1.6.11 -t 1.7.10 -t 1.8.0 -t 2.0.0 -t 3.0.0 -t 4.0.4 -r electron --strip",
-    "prebuild-electron-ia32": "prebuild -t 1.6.11 -t 1.7.10 -t 1.8.0 -t 2.0.0 -t 3.0.0 -t 4.0.4 -r electron -a ia32 --strip",
+    "prebuild-electron": "prebuild -t 1.6.11 -t 1.7.10 -t 1.8.0 -t 2.0.0 -t 3.0.0 -t 4.0.4 -t 5.0.0 -r electron --strip",
+    "prebuild-electron-ia32": "prebuild -t 1.6.11 -t 1.7.10 -t 1.8.0 -t 2.0.0 -t 3.0.0 -t 4.0.4 -t 5.0.0 -r electron -a ia32 --strip",
     "upload": "node ./script/upload.js",
     "postpublish": "git push --follow-tags"
   },

